src/components/BoardReactionMenu/BoardReactionMenu.scss

```diff
@@ -0,0 +1,100 @@
+@import "src/constants/style";
+
+.board-reactions__root {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  bottom: $column__border-width;
+}
+
+.board-reactions__container {
+  width: 374px;
+  height: 64px;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  gap: 8px;
+
+  background-color: $color-white-one;
+  border-radius: $rounded--large;
+  box-shadow: $box-shadow--light;
+
+  padding: 0 $padding--default;
+
+  user-select: none;
+}
+
+.board-reactions__item {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  cursor: pointer;
+}
+
+.board-reactions__reaction {
+  all: unset; // reset button
+  cursor: pointer;
+
+  width: 54px;
+  height: 54px;
+
+  font-size: $text--xl;
+
+  border-radius: $rounded--full;
+
+  span {
+    margin-left: 0.3em; // manually center those stubborn emojis
+  }
```

Comment:
```suggestion
display: flex;
justify-content: center;
align-items: center;
```

Reply:
weird, it works for me:
<img width="572" alt="Bildschirmfoto 2023-10-06 um 09 48 32" src="https://github.com/inovex/scrumlr.io/assets/35272402/505c0e21-f3c4-4ab0-afa7-66ea6763c2fa">

What browser are you using?

This is the code:

```
.board-reactions__reaction {
  all: unset; // reset button
  cursor: pointer;

  width: $reaction-size;
  height: $reaction-size;

  font-size: $text--xl;

  border-radius: $rounded--full;

  display: flex;
  justify-content: center;
  align-items: center;

  &:hover {
    background-color: $hover-background-light;
  }

  &--disabled {
    filter: grayscale(100%);
    cursor: not-allowed;
  }
}
```